src/renderer/containers/Header/index.tsx

```diff
@@ -97,12 +98,14 @@ export const Header: React.FC<HeaderProps> = ({ path, menuItems }) => {
       css={css`
         display: flex;
         justify-content: space-between;
+        ${platformTitleBarStyles}
```

Comment:
Not really a fan of this style in the main view. It makes the header really fat and chonky. Can you maybe reduce the size of the slippi logo or something to make the header less tall?

Reply:
macOS Big Sur styles are chonky and make use of space. Unless there's a specific reason to avoid it I'd prefer keeping it. You also need to do minimum 32px buffer on top with Electron to keep the titlebar dragging working.

Reply:
Looking back over this, I'm still thinking it should stay as-is. Fullscreening the app and having a thinner bar up top gets odd because you can't move the mouse long in that area without the system menubar coming down and covering a chunk of it all. You'd also ideally need to adjust padding different for a fullscreen one since the traffic buttons are in a different spot there.

Electron makes this more annoying; a standard NSToolbar would have the correct behavior built-in to "just work", but I don't think this is worth dealing with.